test/helpers/applyReadConcern.test.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const assert = require('assert');
+const applyReadConcern = require('../../lib/helpers/schema/applyReadConcern');
+const start = require('../common');
+const mongoose = start.mongoose;
+
+describe('applyReadConcern', function() {
```

Comment:
Also add a test case in `query.test.js` that calls `_optionsForExec()` and makes sure that 1) schema-level read concern is applied if user doesn't pass in `readConcern` option, 2) calling `Model.find().setOptions({ readConcern: { level: 'majority' } })` overrides schema-level read concern